frontend/src/lib/lemon-ui/LemonTable/TableCellSparkline.tsx

```diff
@@ -4,9 +4,28 @@ import { offset } from '@floating-ui/react'
 import { Chart, ChartItem, TooltipModel } from 'lib/Chart'
 import { getColorVar } from 'lib/colors'
 import { Popover } from 'lib/lemon-ui/Popover/Popover'
+import React from 'react'
 import { useEffect, useRef, useState } from 'react'
 
-export function TableCellSparkline({ labels, data }: { labels?: string[]; data: number[] }): JSX.Element {
+export interface SparkLineTimeSeries {
+    name: string | null // used in the tooltip
+    color: string // check vars.css for available colors
+    values: number[]
+}
+
+function isSparkLineTimeSeries(data: number[] | SparkLineTimeSeries[]): data is SparkLineTimeSeries[] {
```

Comment:
I think it'd be better for long term maintainability to extract a generic `Sparkline` / `LemonSparkline` component instead of adapting this inner component and re-using it in a different scenario.

Reply:
I don't think there's anything table-specific in this component, so pulled it out into `lib/lemon-ui/` as simply `Sparkline`. Could use a bit of visual tightening but should be useful generally.